<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Color</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>54</width>
           <height>65</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="rb_red">
            <property name="text">
             <string>Red</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_green">
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_blue">
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Primitive</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>59</width>
           <height>88</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="rb_square">
            <property name="text">
             <string>Square</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_circle">
            <property name="text">
             <string>Circle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_star">
            <property name="text">
             <string>Star</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_rhomb">
            <property name="text">
             <string>Rhomb</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_draw">
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
